utils/query.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func ParseQueryResponse(response *ceresdbproto.QueryResponse) ([]map[string]interface{}, error) {
 	if response.SchemaContent == "" {
-		return nil, errors.New("empty schema content")
```

Comment:
Why this change?

Add comments when it will reach here

Reply:
done